src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/jpa/entity/NameEntity.kt

```diff
@@ -41,24 +47,31 @@ class AliasEntity(
   @Column(name = "date_of_birth")
   val dateOfBirth: LocalDate? = null,
 
+  @Column
+  @Enumerated(STRING)
+  val type: NameType,
+
   @Version
   var version: Int = 0,
 ) {
   companion object {
-    private fun from(alias: Alias): AliasEntity? =
+    private fun from(name: Name): NameEntity? =
       when {
-        isAliasPresent(alias.firstName, alias.middleNames, alias.lastName) ->
-          AliasEntity(
-            firstName = alias.firstName,
-            middleNames = alias.middleNames,
-            lastName = alias.lastName,
+        nameExists(name.firstName, name.middleNames, name.lastName) ->
+          NameEntity(
+            title = name.title,
+            firstName = name.firstName,
+            middleNames = name.middleNames,
+            lastName = name.lastName,
+            dateOfBirth = name.dateOfBirth,
```

Comment:
This is strange. I know one of the source systems has this but do we actually want to divorce date of birth from the person?

I can see that it is handy to always have it stored in the same location though